subsys/bluetooth/mesh/Kconfig

```diff
@@ -1375,6 +1375,14 @@ config BT_MESH_PRIV_BEACON_CLI
 
 endif # BT_MESH_PRIV_BEACONS
 
+config BT_MESH_COMP_PST_BUF_SIZE
+	int "Composition Data Page persistence buffer size"
+	default 100
+	help
+	  Buffer used to temporarily hold composition data pages
+	  during flash operations. Should reflect the size of the
+	  largest composition data page present in the application.
```

Comment:
Please add that this buffer is allocated on stack. Also add that after DFU this buffer should be big enough to restore previously stored page, otherwise the page won't be fully restored.

Reply:
Hm. If this is allocated on stack, should stack size be increased either relatively?

Reply:
@alxelax Hmm... not sure.

Reply:
Did you try to pass provisioning procedure with enabled settings in this new changes on real device?